test/instance.ts

```diff
@@ -555,7 +568,7 @@ describe('Instance', () => {
       const gaxOptions = {};
 
       instance.request = (config, callback: Function) => {
-        assert.strictEqual(config.gaxOpts, gaxOptions);
+        assert.deepStrictEqual(config.gaxOpts, gaxOptions);
```

Comment:
```suggestion
        assert.strictEqual(config.gaxOpts, gaxOptions);
```

If after making that change, the test fails, then we would want to keep `deepStrictEqual`, but also set a property on `gaxOptions`.